101-machine-learning-linkedserivce-create/prereqs/nestedtemplates/logicapproleassignments.json

```diff
@@ -0,0 +1,27 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "logicAppName": {
+            "type": "string"
+        }
+    },
+    "variables": {
+        "contributor": "b24988ac-6180-42a0-ab88-20f7382dd24c",
+        "roleDefinitionId": "[resourceId('Microsoft.Authorization/roleDefinitions', variables('contributor'))]",
+        "roleAssignmentName": "[guid(parameters('logicAppName'), resourceGroup().id, variables('roleDefinitionId'))]"
```

Comment:
Use:

"roleAssignmentName": "[guid(resourceId('Microsoft.Logic/workflows', parameters('logicAppName')), resourceGroup().id, variables('roleDefinitionId'))]"

Reply:
will do. I copied this one from Synapse template, you may want to also comment on the original Synapse template.